packages/ERTP/test/unitTests/mathHelpers/test-setMathHelpers.js

```diff
@@ -59,7 +59,7 @@ const runSetMathHelpersTests = (t, [a, b, c], a2) => {
   if (a2 !== undefined) {
     t.throws(
       () => m.make(mockBrand, harden([a, a2])),
-      { message: /value has duplicates/ },
+      { message: /value has duplicate(| key)s/ },
```

Comment:
I don’t think we need another reviewer, but it would be good for @gibson042 or @erights to know that this change created a complication for synchronizing Endo. It would be good for this frustration to have been discovered earlier and for a change to land in Agoric or Endo to iron it out before we attempted integration.

Reply:
> to have been discovered earlier

Would that be done by a process change or new tooling? If the latter, is there a ticket?

Reply:
I have used https://github.com/Agoric/agoric-sdk/pull/7937 on occasion to detect and fix such problems. I think this is similar to some automation @michaelfig already did?